src/GraphQL.NewtonsoftJson/ExecutionResultContractResolver.cs

```diff
@@ -21,6 +21,9 @@ protected override JsonConverter ResolveContractConverter(Type objectType)
             if (typeof(ExecutionResult).IsAssignableFrom(objectType))
                 return new ExecutionResultJsonConverter(_errorInfoProvider);
 
+            if (typeof(GraphQLRequest) == objectType)
```

Comment:
Let's say that `ExecutionResult` is _GraphQL core spec related_ thing . Then `GraphQLRequest` is not _GraphQL core spec related_ thing. It's a _HTTP transport related_ thing.